lighting.py

```diff
@@ -103,6 +106,10 @@ def attributeChangeCallback(
     size: int,
     value: bytes,
 ):
+    
+    # convert bytes to little-endian unsigned integer
+    value = struct.unpack("<I", value)[0]
```

Comment:
If this is changes to uint, should the following code still treat `value` as an array?